src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx

```diff
@@ -12,21 +12,34 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
 * Deploy New Relic APM and infrastructure agents through an automated setup flow 
 * Manage the metrics and logs for all your resources through an interactive dashboard
 
-## What you can monitor [#features]
+## Features [#features]
 
-You can deploy the following agents through the Azure Native New Relic service:
+The features and integrations available for the Azure Native New Relic Service is different from regular, non-Azure New Relic. 
 
-APM agents:
+You can use the following New Relic agents:
+
+**APM agents:**
 
 * .NET
 * Java
 
-Infrastructure agents: 
+**Infrastructure agent for:** 
 
 * Windows
 * Linux 
 
```

Comment:
Other agents and integrations are available through the New Relic platform and can be found through the [I/O integrations](https://newrelic.com/instant-observability/).